Pilot/FacebookService.swift

```diff
@@ -12,6 +12,11 @@ import SwiftyJSON
 
 class FacebookService: NSObject, FileService {
 
+  // Content array
+  var content: [LocalFile] = []
+
+  var folderMonitor: FolderMonitor!
```

Comment:
I store it as a reference in the service classes in case we need to use it again. It felt a little to just let it go in the background without any control for the life of the program.